app/views/pulls/_merge.html.erb

```diff
@@ -22,6 +22,24 @@
     <% else %>
       <strong><%= l(("label_merge_" + pull.merge_status + "_title").to_sym) %></strong>
       <em class="info"><%= l(("label_merge_" + pull.merge_status + "_help").to_sym) %></em>
+
+      <% if pull.merge_status == 'conflicts_detected' && @conflicting_files&.length %>
```

Comment:
With the method `Pull#conflicting_files` proposed above, you'll be able to simply write this:

```suggestion
      <% unless pull.conflicting_files.empty? %>
```